tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -113,18 +126,23 @@ def test_generate_jwt_from_jwks(mocker):
     mock_get.return_value = MockRequest(data={"keys": [jwk]})
     assert generate_jwt_from_jwks() is not None
 
+
 def test_validate_nonce_and_state():
     """Test nonece and state validation."""
     assert validate_nonce_and_state("x", "y", "x", "y") is True
     assert validate_nonce_and_state("x", "z", "x", "y") is False
     assert validate_nonce_and_state("x", "y", "y", "x") is False
     assert validate_nonce_and_state("x", "z", "y", "y") is False
 
+
+@pytest.mark.skipif(not test_private_key, reason="No test private key set")
```

Comment:
Seems like it would be better for the test to fail if the test_private_key isn't there